WeakEvent/WeakEventSource.cs

```diff
@@ -8,9 +8,7 @@
 namespace WeakEvent
 {
     public class WeakEventSource<TEventArgs>
-#if NET40
         where TEventArgs : EventArgs
-#endif
```

Comment:
This should be changed to `#if NET40 || NET35`, not removed. .NET 4.5 and .NET Standard no longer have a constraint on `TEventArgs`.

Reply:
OK, I'll revert this. I misunderstood the intention.